samples/dnn/scene_text_detection.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * Real-time Scene Text Detection with Differentiable Binarization
+ * Copyright (C) 2020, The patent is owned by <Huazhong University of Science and Technology>
+ * Author List:
+ *      Minghui Liao    <Huazhong University of Science and Technology>
+ *      Zhaoyi Wan      <Megvii>
+ *      Cong Yao        <Megvii>
+ *      Kai Chen        <Shanghai Jiao Tong University>
+ *      Xiang Bai       <Huazhong University of Science and Technology>, Corresponding author
+ *
+ * This script is written by Wenqing Zhang <Huazhong University of Science and Technology>.
+ *
+ * The code has been contributed to OpenCV under the terms of Apache 2 license:
+ * https://www.apche.org/licenses/LICENSE-2.0
+*/
+
+#include <iostream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/dnn/dnn.hpp>
+
+using namespace cv;
+using namespace cv::dnn;
+
+std::string keys =
+        "{ help  h                          | | Print help message. }"
+        "{ inputImage i                     | | Path to an input image. Skip this argument to capture frames from a camera. }"
+        "{ device d                         | 0 | camera device number. }"
+        "{ modelPath mp                     | | Path to a binary .onnx file contains trained DB detector model.}"
```

Comment:
just curious -- where do we find the model (link) ?

Reply:
Maybe I can upload models to Google Drive.
I am not sure about it, and I will ask my mentor.